config.plist/alder-lake.md

```diff
@@ -0,0 +1,793 @@
+  * Used for Slide variable calculation. However the necessity of this quirk is determined by `OCABC: Only N/256 slide values are usable!` message in the debug log. If the message `OCABC: All slides are usable! You can disable ProvideCustomSlide!` is present in your log, you can disable `ProvideCustomSlide`.
+* **RebuildAppleMemoryMap**: YES
+  * Generates Memory Map compatible with macOS, can break on some laptop OEM firmwares so if you receive early boot failures disable this.
+* **ResizeAppleGpuBars**: -1
+  * Will reduce the size of GPU PCI Bars if set to zero when booting macOS.
+  * Setting other PCI Bar values is possible with this quirk, though can cause instabilities
+  * This quirk being set to zero is only necessary if Resizable GPU Bar Support is enabled in your firmware.
+* **SetupVirtualMap**: NO
+  * Fixes SetVirtualAddresses calls to virtual addresses, however broken due to Comet Lake's memory protections. ASUS, Gigabyte and AsRock boards will not boot with this on.
+* **SyncRuntimePermissions**: YES
+  * Fixes alignment with MAT tables and required to boot Windows and Linux with MAT tables, also recommended for macOS. Mainly relevant for RebuildAppleMemoryMap users.
+
+:::
+
+## DeviceProperties
+
+![DeviceProperties](../images/config/config.plist/cometlake/DeviceProperties.png)
+
+### Add
+
+Sets device properties from a map.
+
+::: tip PciRoot(0x0)/Pci(0x1C,0x3)/Pci(0x0,0x0)
+
+This entry relates to Intel's I225-V 2.5GBe controller found on higher end Comet Lake boards, what we'll be doing here is tricking Apple's I225LM driver into supporting our I225-V network controller:
+
+| Key | Type | Value |
+| :--- | :--- | :--- |
+| device-id | Data | `F2150000` |
+
+* **Note**: If your board didn't ship with the Intel I225 NIC, there's no reason to add this entry.
+* **Note 2**: If you get a kernel panic on the AppleIntelI210Ethernet kext, your NIC's path is likely `PciRoot(0x0)/Pci(0x1C,0x4)/Pci(0x0,0x0)`
+
+:::
+
+::: tip PciRoot(0x0)/Pci(0x1b,0x0)
+
+`layout-id`
+
+* Applies AppleALC audio injection, you'll need to do your own research on which codec your motherboard has and match it with AppleALC's layout. [AppleALC Supported Codecs](https://github.com/acidanthera/AppleALC/wiki/Supported-codecs).
+* You can delete this property outright as it's unused for us at this time
+
+For us, we'll be using the boot-arg `alcid=xxx` instead to accomplish this. `alcid` will override all other layout-IDs present. More info on this is covered in the [Post-Install Page](https://dortania.github.io/OpenCore-Post-Install/)
+
+:::
+
+### Delete
+
+Removes device properties from the map, for us we can ignore this
+
+## Kernel
+
+![Kernel](../images/config/config-universal/kernel-modern-XCPM.png)
+
+### Add
+
+Here's where we specify which kexts to load, in what specific order to load, and what architectures each kext is meant for. By default we recommend leaving what ProperTree has done, however for 32-bit CPUs please see below:
```

Comment:
> however for 32-bit CPUs please see below:

Huh

Reply:
This was copied from the Comet Lake guide lol. So I guess it's in that and maybe a couple others. I'll fix it everywhere it appears as soon as I get the chance.

Reply:
Resolved in 1a2aad8